apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -302,7 +302,25 @@ export const include: Includeable[] = [
   },
   { model: Notification, as: 'notifications' },
   { model: Case, as: 'mergeCase' },
-  { model: Case, as: 'mergedCases', separate: true },
+  {
+    model: Case,
+    as: 'mergedCases',
+    where: { state: CaseState.COMPLETED },
+    include: [
+      {
+        model: CaseFile,
+        as: 'caseFiles',
+        required: false,
+        where: { state: { [Op.not]: CaseFileState.DELETED }, category: null },
```

Comment:
Don't we need to include other categories here? The category `CASE_FILE` is not needed as those files are included in the generated case files records, but I would think other categories relevant to indictments should be included.

Reply:
Yes you're right, I removed the category filter in my other PR but this comment made me realize I need to filter out CASE_FILE so I'll add that in there